web-ts/src/routes/sprungbrett/SprungbrettOfferPage.tsx

```diff
@@ -0,0 +1,37 @@
+import React, { ReactElement } from 'react'
```

Comment:
I'm not sure whether we discussed this somewhere already, but would it be possible to remove this one layer of directories here?

To have just a directory with an index.ts: `src/routes/SprungbrettOfferPage/index.ts`
or just have a single js file per route? `src/routes/SprungbrettOfferPage.ts?

Reply:
Done in #311, you just reviewed it :D

Reply:
oh oke :D quite a lot of refactoring :P